crm_request/models/request.py

```diff
@@ -17,8 +17,9 @@ class CrmClaim(models.Model):
         help="The destination email address that the contacts used.")
     code = fields.Char(string='Number')
     claim_type = fields.Many2one(string='Type')
-    user_id = fields.Many2one(string='Assign to')
-    stage_id = fields.Many2one(group_expand='_read_group_stage_ids')
+    user_id = fields.Many2one(string='Assign to', track_visibility=False)
+    stage_id = fields.Many2one(group_expand='_read_group_stage_ids',
+                               track_visibility=False)
```

Comment:
I don't understand why your remove this. It could be useful to be notified when the stage of request changed.